EKF/ekf.cpp

```diff
@@ -216,6 +213,9 @@ bool Ekf::initialiseFilter()
 		// reset the output predictor state history to match the EKF initial values
 		alignOutputFilter();
 
+		// initialise the state covariance matrix now we have starting values for all lthe states
+		initialiseCovariance();
```

Comment:
@priseborough If we call this function here, aren't we then overriding the variances set by increaseQuatYawErrVariance() [here](https://github.com/PX4/ecl/pull/739/files#diff-0871d0235139c63531138e2793b76de7L202) ?

Reply:
@priseborough Would it make sense to have increaseQuatYawErrVariance() as part of initialiseCovariance(). Then we are always sure it gets called at the right moment.

Reply:
See https://github.com/PX4/ecl/pull/739/commits/0d12f1bdde6431dc32c2681049f8bffd36b9be21

Safer now to move it before the yaw variance is modified.